templates/.github/CONTRIBUTING.md

```diff
@@ -1,3 +1,9 @@
+<!--
+ - SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
```

Comment:
I think a somewhat standard thing is to align the dash with the last dash of the first line 🤔.